bascule/jws.go

```diff
@@ -0,0 +1,24 @@
+package bascule
+
+import (
+	"github.com/SermoDigital/jose/jws"
+)
+
+// JWSParser parses raw Tokens into JWS objects
+type JWSParser interface {
+	ParseJWS([]byte) (jws.JWS, error)
+}
+
+type defaultJWSParser int
```

Comment:
`type defaultJWSParser struct{}` is preferred over this approach.

Reply:
changed this, thanks!